content/en/tracing/profiler/getting_started.md

```diff
@@ -256,6 +256,88 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 [6]: /tracing/visualization/#services
 [7]: /tracing/guide/setting_primary_tags_to_scope/#environment
 {{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+<div class="alert alert-warning">
+Datadog Ruby Profiler is currently in public beta. We strongly advise evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The Datadog Profiler requires MRI Ruby 2.1+ and **CPU-time profiles are only available on Linux platforms**. To begin profiling applications:
```

Comment:
Not obvious if there are other things that will work on non-linux platforms.

Reply:
I think this was also reflected on Marco's feedback. Perhaps we should state here what users should expect on other platforms, e.g.

> The Datadog Profiler requires MRI Ruby 2.1+. Wall time profiling is available for users on every platform (including macOS and Windows), but CPU-time profiles are currently only available on Linux platforms.

Reply:
I've also created https://github.com/DataDog/dd-trace-rb/pull/1517 to improve the message we log to the console to include this "...but this is what you should expect" addendum as well.